bundles/AdminBundle/Resources/public/js/pimcore/asset/asset.js

```diff
@@ -32,6 +32,10 @@ pimcore.asset.asset = Class.create(pimcore.element.abstract, {
         try {
             this.data = Ext.decode(response.responseText);
 
+            if (!this.data.success && this.options && this.options.ignoreNotFoundError) {
```

Comment:
this breaks memorized asset tab in general as `success = true` is not passed when image is found
![image](https://user-images.githubusercontent.com/5137917/130559430-b4d16ffb-48cf-4fd5-b546-1c018b171ffa.png)
```suggestion
            if (this.data.success === false && this.options && this.options.ignoreNotFoundError) {
```